.github/workflows/self-test.yml

```diff
@@ -42,6 +42,10 @@ jobs:
         # definition in env does not work since $GITHUB_WORKSPACE seems not to
         # be accessible
         export RIOTBASE="$GITHUB_WORKSPACE/RIOT"
+        export LORAWAN_APP_KEY='${{ secrets.CI_TTN_APPKEY }}'
+        export LORAWAN_NWK_SKEY='${{ secrets.CI_TTN_NWKSKEY_ABP }}'
+        export LORAWAN_APP_SKEY='${{ secrets.CI_TTN_APPSKEY_ABP }}'
+        export LORAWAN_DL_KEY='${{ secrets.CI_TTN_APPID_KEY }}'
```

Comment:
I think those needs to be added to the release-tests workflow in RIOT-OS/RIOT ;-). The workflow here only tests the test framework itself.

Reply:
ok, I do that :)

Reply:
done